### DISCUSSION TITLE
Deprecated watchdoh option

### DISCUSSION THREAD

**Discussion by mdvorak** [frenck/action-app-linter#160] [Q&A]
In https://github.com/frenck/action-addon-linter/pull/139 was added error for `watchdog` config, but I can't find any information, that it is actually being deprecated.
Does anyone have any information about that?
Thanks

**Comment by frenck** [comment]
The option is obsolete but currently kept for backward compatibility.

Leveraging HEALTHCHECK is much more flexible and provides instant feedback to Home Assistant if something happens.

../Frenck
